app/controllers/allocate_controller.rb

```diff
@@ -0,0 +1,49 @@
+class AllocateController < ApplicationController
+  before_action :authenticate_user
+
+  def show
+    response = OffenderService.new.get_offender(nomis_offender_id)
+    @prisoner = response.data
+
+    @recommended_pom = @prisoner.current_responsibility
+
+    pom_response = StaffService.new.get_prisoner_offender_managers(caseload)
+    @recommended_poms, @not_recommended_poms = pom_response.data.partition { |pom|
+      pom.position_description.include?(@recommended_pom)
+    }
+  end
+
+  def new
+    @prisoner = prisoner
+
+    pom_response = StaffService.new.get_prisoner_offender_managers(caseload)
+    @pom = pom_response.data.select { |p| p.staff_id == nomis_staff_id }.first
```

Comment:
We might want to create a Trello card to track the need for a 'get_pom(staff_id)' entry in the service.